flutter/test/sentry_flutter_util.dart

```diff
@@ -9,60 +9,53 @@ import 'package:sentry_flutter/src/version.dart';
 
 import 'mocks.dart';
 
-FutureOr<void> Function(SentryOptions) getConfigurationTester({
-  bool isIOS = false,
-  bool isWeb = false,
-  bool isAndroid = false,
+FutureOr<void> Function(SentryFlutterOptions) getConfigurationTester({
+  required List<Type> shouldHaveIntegrations,
+  required List<Type> shouldNotHaveIntegrations,
+  required bool hasFileSystemTransport,
 }) =>
     (options) async {
-      assert(options is SentryFlutterOptions);
-
       options.dsn = fakeDsn;
 
       expect(kDebugMode, options.debug);
```

Comment:
I think this should now probably go into a dedicated test.

Reply:
makes sense